elements/pfe-cta/src/pfe-cta.scss

```
@@ -377,5 +379,22 @@ $pfe-cta--Color--fallback: #003366;
     --pfe-cta--Color--hover:            #{pfe-color(ui-complement)};
 }
 
+/// ===========================================================================
+/// WIND CTA VARIANTS
+/// ===========================================================================
 
-
+:host([pfe-priority="secondary"][pfe-variant="wind"]) {
+  --pfe-cta--BackgroundColor:           transparent;
+  --pfe-cta--BorderColor:               var(--pfe-theme--surface--border, #d2d2d2);
```

Use the pfe-color mixin here:

```suggestion
  --pfe-cta--BorderColor:               #{pfe-color(surface--border)};
```